llvm/include/llvm/CodeGen/TargetLowering.h

```diff
@@ -5616,6 +5616,10 @@ class TargetLowering : public TargetLoweringBase {
     return true;
   }
 
+protected:
+  // Simple interface for targets without a Module dependency
```

Comment:
There probably aren't that many overrides here? Maybe easiest to just always override the version with the Module&?

Reply:
Yup, that makes things look cleaner. Good idea.